addon/pods/components/frost-tab/component.js

```diff
@@ -1,21 +1,55 @@
 import Ember from 'ember'
 import layout from './template'
-import FrostTabs from 'ember-frost-tabs/pods/components/frost-tabs/component'
+// import FrostTabs from 'ember-frost-tabs/pods/components/frost-tabs/component'
+import { PropTypes } from 'ember-prop-types'
+
+const {
+  Component
+} = Ember
+
+export default Component.extend({
+  // == Component properties ==================================================
 
-export default Ember.Component.extend({
   layout: layout,
-  classNames: ['content'],
-  classNameBindings: ['isSelected::hidden', 'tabClassNames'],
+  classNames: ['frost-tab'],
+  classNameBindings: ['tabClassNames'],
 
-  frostTabs: null,
+  // == State properties ======================================================
+
+  propTypes: {
+    id: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired,
+    content: PropTypes.oneOfType([
+      PropTypes.EmberObject,
+      PropTypes.object
+    ]).isRequired,
+    disabled: PropTypes.bool,
+    // Set by the parent component
+    hook: PropTypes.string,
+    targetOutlet: PropTypes.string.isRequired,
+    selectedTab: PropTypes.string,
+    onChange: PropTypes.func
```

Comment:
required?  Also if we're updating the interface would we want to change this to something like `onSelect`?